query-algebrizer/src/cc.rs

```diff
@@ -83,6 +87,12 @@ impl<T> OptionEffect<T> for Option<T> {
     }
 }
 
+fn unit_type_set(t: ValueType) -> HashSet<ValueType> {
```

Comment:
Don't you wish Rust made this easier?  Like `set.with(...)` or `Set::of{1,2,3}(...)`.